yamux/Cargo.toml

```diff
@@ -7,7 +7,6 @@ description = "Multiplexer over reliable, ordered connections"
 keywords = ["network", "protocol"]
 categories = ["network-programming"]
 repository = "https://github.com/paritytech/yamux"
-readme = "README.md"
```

Comment:
Should it be "../README.md" then?

Reply:
My bad. Probably. I thought crates.io can infer it from the repository URL but it doesn't. Too late for the v0.11.0 release. @thomaseizinger can you provide a patch for future releases?